src/main/java/uk/gov/justice/digital/converter/dms/DMS_3_4_6.java

```diff
@@ -81,6 +82,8 @@ public static Optional<Operation> getOperation(String operation) {
         }
     }
 
+    public static DMS_3_4_6.Operation[] cdcOperations = { Insert, Update, Delete };
```

Comment:
I think it's better to have this as a property on the `Operation` enum. See my comment re. the set further down.